dev-packages/node-integration-tests/suites/public-api/LocalVariables/local-variables-out-of-app-default.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-unused-vars */
+
+const Sentry = require('@sentry/node');
+// const { loggingTransport } = require('@sentry-internal/node-integration-tests'); is throwing error that package not found, so using relative path
```

Comment:
This shouldn't be happening, are you running `yarn build` in the root of the repo? You might have to run `yarn clean` and delete node modules to get stuff in a good state.

Reply:
I wonder if the issue is `yarn workspace`, if you `cd` into the directory and run the tests there directly do you get the same issues? I'm unfortunately not able to reproduce.